Makefile

```diff
@@ -658,7 +664,7 @@ check-symbols: startup_files libc
 
 	# Check that the computed metadata matches the expected metadata.
 	# This ignores whitespace because on Windows the output has CRLF line endings.
-	diff -wur "$(CURDIR)/expected/$(MULTIARCH_TRIPLE)/$(THREAD_MODEL)" "$(SYSROOT_SHARE)"
+	diff -wur "$(CURDIR)/expected/$(TARGET_TRIPLE)/$(THREAD_MODEL)" "$(SYSROOT_SHARE)"
```

Comment:
Oh I see.   Now that we have the thread model as part of the triple we don't need the extra directory level here.. we can just remove `/$(THREAD_MODEL)`

Reply:
Good point, I can change that.